Add content / references to Parity Devops Guide (#17)
w3f/educhain: docs/collator.md

- **Candidates** - Accounts that have registered for becoming collators typically by bonding a deposit. These accounts receive authorization to collate if they meet the requirements set by the network. This ensures a permissionless way of registering to becoming a collator.
- **Invulnerables** - Accounts that are *guaranteed* to participate in block production, irrespective of the boding requirements. If the collator selection mechanism is Aura, they will participate in block production round-robin style.

Candidates can add or remove themselves from collation on a live network. **Invulnerables**, however, can only be added or removed through an account with sudo or root privileges. They are also usually specified in the chain spec as the "bootnodes".  It is wise to add at least one collator in your chain_spec - and one that you can start easily. That way you can always gurantee a collator that can produce blocks.

> ## Node Roles & Resources
> 
> For simplicity, EduChain has combined multiple roles into a single instance.  However, for more robust networks, it is recommended to split your operations into several instances so as to not overburden a single piece of architecture.
> 
> For more information and details about different roles, refer to the [Parity Devops Guide.](https://paritytech.github.io/devops-guide/deployments/roles.html)

Reviewer: ```suggestion
## Node Roles & Resources

For simplicity, EduChain runs a single node with multiple roles. However, for robustness, it is recommended that you split the network operations into several instances of nodes.

For more information and details about different roles, refer to the [Parity DevOps Guide.](https://paritytech.github.io/devops-guide/deployments/roles.html)
```

pub const SESSION2: &str = "0x1e673715db64783eadc6ca927e493ded30f2447efff0f6d5d84578e823f86374";
```

> For an explanation on the types of keys, their specific types, and how to generate them, refer to the [Parity Devops documentation.](https://paritytech.github.io/devops-guide/explanations/keys_accounts.html)

Reviewer: ```suggestion
For an explanation on the types of keys, their specific types, and how to generate them, refer to the [Parity DevOps documentation.](https://paritytech.github.io/devops-guide/explanations/keys_accounts.html)
```
